README.md

```diff
@@ -157,30 +157,28 @@ Last Updated: `2023-06-22 18:25:15.707233069 +0000 UTC m=+0.079645794`
           terraform show -json tfplan-${{ matrix.workspace }} > tfplan-${{ matrix.workspace }}.json
         shell: bash
       - name: Archive Plans
-        uses: actions/upload-artifact@v3
+        uses: u21-public/terraform-plan-analyzer/actions/archive-plans@v0.6.0
         with:
-          name: tfplans
-          path: |
-            tfplan-{{ matrix.workspace }}.json
+          plan_name: "tfplan-${{ matrix-planname }}"
 ```
 2) Unarchieve plans and analyze them 
 ```
   analyze-plans:
-    runs-on: ubuntu-latest
-    needs:[run-service-plan]
+    needs: [run-service-plan]
+    if: |
+      always()
+    runs-on: ${{ inputs.runner }}
+    name: Analyze Plans
     steps:
-      - name: download plans
-        uses: actions/download-artifact@v3
+      - name: Confirm Passing plans
+        if: ${{ needs.run-service-plan.result != 'success' }}
+        run: exit 1
+      - name: Analyze Terraform Plans
+        uses: u21/terraform-plan-analyzer/actions/analyze-plans@v1.2.0
+        uses: u21-public/terraform-plan-analyzer/actions/analyze-plans@v0.6.0
```

Comment:
L177 should be removed right? we can only have one `uses` key?

Reply:
sigh. copy pasta on my end